### PR TITLE
[Doc][Improve] add support chinese for some sinks

### PR DESCRIPTION
**Pull request by corgy-w**
### Check list

-  If any new Jar binary package adding in your PR, please add License Notice according
  [[New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)

-  If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs

-  If you are contributing the connector code, please check that the following files are updated:

  1. Update change log that in connector document. For more details you can refer to [[connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [[plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties)](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [[seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)

-  Update the [`[release-note](https://github.com/apache/seatunnel/blob/dev/release-note.md)`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by corgy-w** [comment]
@baicie Hello, I ask a question. Once I get to this step, how do I fix it and move on to the next step? Please excuse me, this is my first time contributing to open source. TKS！

**Comment by baicie** [comment]
> @baicie你好，我问一个问题。到达这一步后，我该如何修复它并继续下一步？请原谅，这是我第一次为开源做出贡献。 TKS！
open actions in you fork repo
image

**Comment by Hisoka-X** [comment]
cc @zhilinli123

File: docs/zh/connector-v2/sink/Console.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## 支持的引擎
+
+> Spark<br/>
+> Flink<br/>
+> SeaTunnel Zeta<br/>
+
+## 描述
+
+用于向控制台发送数据。两者都支持流和批处理模式。
```
Comment (EricJoy2048) [inline docs/zh/connector-v2/sink/Console.md]:
```suggestion
接收Source端传入的数据并打印到控制台。支持批同步和流同步两种模式。
```

**Pull request closed without merging**